docs/kubernetes.md

```diff
@@ -32,6 +32,34 @@ In the image above, you can see the following parts:
 
 All VMs are in the same private VNET and are fully accessible to each other.
 
+
+## Optional: Enable network policy enforcement using Calico

```

Comment:
I see an issue with the location of this section.  At this point in the walkthrough, the Kubernetes cluster is already deployed (i.e the user has followed the Deployment section above), which means if they get to this point and decide they want Calico they need to tear down the cluster they've deployed and create a new one.

Could we include this as an optional step in the `## Deployment` section above to make it more clear to users that this is an option?